### PR TITLE
Removed EBI layer on Dumps-related pipelines

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
Ticket [ENSCOMPARASW-2486](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-2486) (part 5/7).

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm [hunk 1]
```diff
@@ -17,9 +17,10 @@ limitations under the License.
 
 =cut
 
-=head1 SYNOPSIS
+=head1 DESCRIPTION
 
-Pipeline to dump conservation scores as bedGraph and bigWig files
+    This is a partial PipeConfig for most part of the pipeline that dumps the
+    conservation scores as bedGraph and bigWig files.
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm]: I'm not sure about this description - can we change it?
> This PipeConfig contains the core analyses required to dump the conservation scores as bedGraph and bigWig files.

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConstrainedElements.pm [hunk 2]
```diff
@@ -17,14 +17,10 @@ limitations under the License.
...
-=head1 SYNOPSIS
+=head1 DESCRIPTION
 
-Initialise the pipeline on compara1 and dump the constrained elements of mlss_id 836
-found at cc21_ensembl_compara_86 on compara5
-
-  init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpConstrainedElements_conf -compara_db mysql://ensro@compara5/cc21_ensembl_compara_86 -mlss_id 836 -host compara1
-
-Dumps are created in a sub-directory of --export_dir, which defaults to scratch109
+    This is a partial PipeConfig for most part of the pipeline that dumps the
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConstrainedElements.pm]:
```suggestion
    This PipeConfig contains the core analyses required to dump the
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpMultiAlign.pm [hunk 3]
```diff
@@ -19,6 +19,8 @@ limitations under the License.
 
 =head1 SYNOPSIS
 
+    This is a partial PipeConfig for most part of the pipeline that dumps the
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpMultiAlign.pm]:
```suggestion
    This PipeConfig contains the core analyses required to dump the
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm [hunk 4]
```diff
@@ -26,14 +26,15 @@ Bio::EnsEMBL::Compara::PipeConfig::DumpHomologiesByMLSS_conf
...
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpPerMLSSHomologies_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -division $COMPARA_DIV -member_type ncrna -clusterset_id murinae
 
     By default the pipeline dumps the database named "compara_curr" in the
     registry, but a different database can be selected with -rel_db
 
 =head1 DESCRIPTION
 
-    This pipeline dumps all the gene-trees and homologies under #base_dir#
+This pipeline dumps all the gene-trees and homologies under #base_dir#.
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm]: Not worth fixing, but in some cases you remove leading whitespace and in some, you add it. Not sure what the rules are ;)

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpSpeciesTrees.pm [hunk 5]
```diff
@@ -19,15 +19,12 @@ limitations under the License.
...
-
-=head1 SYNOPSIS
-
-    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpSpeciesTrees_conf -compara_db <url_of_the_compara_db>
+Bio::EnsEMBL::Compara::PipeConfig::Parts::DumpSpeciesTrees_conf
 
 =head1 DESCRIPTION  
 
-Dumps all the species-trees from the database
+    This is a partial PipeConfig for most part of the pipeline that dumps all
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpSpeciesTrees.pm]:
```suggestion
    This PipeConfig contains the core analyses required to dump all
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpTrees.pm [hunk 6]
```diff
@@ -22,18 +22,12 @@ limitations under the License.
...
-    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpTrees_conf -host compara1 -member_type ncrna -clusterset_id murinae
-
-    By default the pipeline dumps the database named "compara_curr" in the
-    registry, but a different database can be selected with -rel_db
+Bio::EnsEMBL::Compara::PipeConfig::Parts::DumpTrees_conf
 
 =head1 DESCRIPTION
 
-    This pipeline dumps all the gene-trees and homologies under #base_dir#
+    This is a partial PipeConfig for most part of the pipeline that dumps all
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpTrees.pm]:
```suggestion
    This PipeConfig contains the core analyses required to dump all
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm [hunk 7]
```diff
@@ -17,9 +17,10 @@ limitations under the License.
 
 =cut
 
-=head1 SYNOPSIS
+=head1 DESCRIPTION
 
-Pipeline to dump conservation scores as bedGraph and bigWig files
+    This is a partial PipeConfig for most part of the pipeline that dumps the
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm]:
```suggestion
    This PipeConfig contains the core analyses required to dump the
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm (hunk 1 above)
Comment (muffato) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/DumpConservationScores.pm]: constrained elements as bigBed and conservation scores as bigWig

File: modules/Bio/EnsEMBL/Compara/PipeConfig/DumpConservationScores_conf.pm [hunk 8]
```diff
@@ -19,7 +19,13 @@ limitations under the License.
...
 =head1 SYNOPSIS
 
-Pipeline to dump conservation scores as bedGraph and bigWig files
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpConservationScores_conf -host mysql-ens-compara-prod-X -port XXXX \
+        -compara_db $(mysql-ens-compara-prod-X details url ${USER}_mammals_epo_low_coverage_${CURR_ENSEMBL_RELEASE}) \
+        -mlss_id XXXX
+
+=head1 DESCRIPTION
+
+    Pipeline to dump conservation scores as bedGraph and bigWig files.
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/DumpConservationScores_conf.pm]: bedGraph files are temporary. Only the bigWig are put on the FTP

File: modules/Bio/EnsEMBL/Compara/PipeConfig/DumpConstrainedElements_conf.pm [hunk 9]
```diff
@@ -19,12 +19,13 @@ limitations under the License.
...
-found at cc21_ensembl_compara_86 on compara5
+    init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpConstrainedElements_conf -host mysql-ens-compara-prodX -port XXXX
+        -compara_db $(mysql-ens-compara-prod-X details url ${USER}_mammals_epo_low_coverage_${CURR_ENSEMBL_RELEASE}) \
+        -mlss_id XXXX
 
-  init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::DumpConstrainedElements_conf -compara_db mysql://ensro@compara5/cc21_ensembl_compara_86 -mlss_id 836 -host compara1
+=head1 DESCRIPTION
 
-Dumps are created in a sub-directory of --export_dir, which defaults to scratch109
+    Pipeline to dump the contrained elements as BED and BigBED files.
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/DumpConstrainedElements_conf.pm]: Bed files are only temporary. Users will get bigBed files

File: modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm (hunk 4 above)
Comment (muffato) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm]: It's indeed confusing. From the other reviews I had the feeling you wanted to remove the leading whitespace but in this PR you've kept it in several files

Comment (JAlvarezJarreta) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/DumpHomologiesByMLSS_conf.pm]: Sorry! Will try to find all that I have left and remove them to set some kind of consistency among myself at least.

**Pull request merged**